### ISSUE TITLE
Problem with VerifySignature Wehbook Request Header

### ISSUE THREAD

**Issue by oladejihenry** [digikraaft/laravel-paystack-webhooks#6]
When i try running the webhook i keep getting the AccessDeniedException and for some reason i believe the request header here might be the problem
```
 if ((! $request->isMethod('post')) || ! $request->header('HTTP_X_PAYSTACK_SIGNATURE', null)) {
            throw new AccessDeniedHttpException("Invalid Request");
        }
```
Because when i did this 
```
$request->header('x-paystack-signature');
```

It works, kindly look into this please. Also my credentials are correct

Also the WebhooksController needs to be changed to this

```
public function handleWebhook(Request $request): \Symfony\Component\HttpFoundation\Response
    {
        $payload = json_decode($request->getContent(), true);
        $method = 'handle'.Str::studly(str_replace('.', '_', $payload['event']));

        WebhookReceived::dispatch($payload);

        if (method_exists($this, $method)) {
            $response = $this->{$method}($payload);

            WebhookHandled::dispatch($payload);

            return $response ?? $this->successMethod();
        }

        return $this->missingMethod();
    }
```

**Comment by timoladoyinbo** [comment]
Hi @oladejihenry I'll have a look at my earliest. It may be that Paystack has changed what to expect from the header.

**Comment by timoladoyinbo** [comment]
Hi @oladejihenry this is now fixed and tagged as [v2.1.1](https://github.com/digikraaft/laravel-paystack-webhooks/releases/tag/v2.1.1)
